client/signup/steps/plans/tabbed-plans.js

```diff
@@ -0,0 +1,512 @@
+
+	const handleUpgradeButtonClick = ( productSlug, productId ) => {
+		const args = productSlug === null ? null : { product_slug: productSlug, product_id: productId };
+		onUpgradeClick( args );
+	};
+
+	useEffect( () => {
+		const planFilter =
+			selectedTab === 'Professional'
+				? [ 'Premium', 'Business', 'eCommerce' ]
+				: [ 'Free', 'Personal' ];
+
+		const displayedPlans = planProperties
+			.filter( ( plan ) => planFilter.includes( plan.planName ) )
+			.filter( ( plan ) => plan.termLength === termLength || plan.termLength === null );
+
+		// Remove free plan if the user has selected a domain
+
+		setFeatureComparison( featureComparisonData[ selectedTab ] );
+		setPlanDetails( displayedPlans );
+	}, [ selectedTab, planProperties, termLength ] );
+
+	return (
+		<>
+			<TabWrapper>
+				<Tabs>
+					{ tabList.map( ( item, index ) =>
+						item === selectedTab ? (
+							<SelectedTab
+								key={ `tab${ index }` }
+								className={ `tabbed-plans__tab-${ index + 1 }` }
+								onClick={ toggleTab }
+							>
+								{ item }
+							</SelectedTab>
+						) : (
+							<Tab
+								key={ `tab${ index }` }
+								className={ `tabbed-plans__tab-${ index + 1 }` }
+								onClick={ toggleTab }
+							>
+								{ item }
+							</Tab>
+						)
+					) }
+				</Tabs>
+			</TabWrapper>
+			<Grid className="tabbed-plans__grid-container">
+				<TermToggles>
+					<RadioButton>
+						<input
+							type="radio"
+							checked={ termLength === 'monthly' ? 'checked' : '' }
+							name="monthly"
+							onClick={ toggleTerm }
+						/>
+						<Checkmark></Checkmark>
+						<span>{ translate( 'Monthly' ) }</span>
+					</RadioButton>
+					<RadioButton>
+						<input
+							type="radio"
+							checked={ termLength === 'yearly' ? 'checked' : '' }
+							name="monthly"
+							onClick={ toggleTerm }
+						/>
+						<Checkmark></Checkmark>
+						<span>{ translate( 'Yearly' ) }</span>
+					</RadioButton>
+				</TermToggles>
+
+				{ selectedTab === 'Professional' && <FeaturedPlan /> }
+
+				{ planDetails &&
+					planDetails.map( ( item, index ) => (
+						<React.Fragment key={ `planDetails${ index }` }>
+							<PlanHeader
+								key={ `planHeader${ index }` }
+								className={ `tabbed-plans__header-${ index + 1 }` }
+							>
+								{ translate( 'WordPress %(planName)s', {
+									args: { planName: item.planName },
+									comment: 'For example, WordPress Business',
+								} ) }
+							</PlanHeader>
+							<PlanPrice
+								key={ `planPrice${ index }` }
+								className={ `tabbed-plans__price-${ index + 1 }` }
+							>
+								{ formatCurrency( item.rawPrice, currencyCode, { stripZeros: true } ) }
+							</PlanPrice>
+							<TermDescription className={ `tabbed-plans__term-desc-${ index + 1 }` }>
+								{ item.termLength === 'yearly' && (
+									<>
+										<span>
+											{ translate( 'per month billed as %(annualPrice)s annually', {
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **24** times:
`translate( 'per month, billed as %s annually' )` *ES Score: 6*